crates/ibc/src/core/ics24_host/identifier.rs

```diff
@@ -194,6 +194,12 @@ impl Default for ClientId {
     }
 }
 
+impl From<ClientId> for String {
```

Comment:
Nit: Can we use `derive_more::Into` instead?

Reply:
https://github.com/cosmos/ibc-rs/pull/208/commits/43b8589fa6211ed50f4025579bcd5912f2becab1